packages/block-library/src/latest-posts/edit.js

```diff
@@ -227,68 +232,130 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 	const hasPosts = !! latestPosts?.length;
 	const inspectorControls = (
 		<InspectorControls>
-			<PanelBody title={ __( 'Post content' ) }>
-				<ToggleControl
-					__nextHasNoMarginBottom
+			<ToolsPanel
+				label={ __( 'Post content' ) }
+				resetAll={ () =>
+					setAttributes( {
+						displayPostContent: false,
+						displayPostContentRadio: 'excerpt', // Set default value for the radio button
+						excerptLength: MIN_EXCERPT_LENGTH, // Default length for the excerpt
```

Comment:
```suggestion
						displayPostContentRadio: 'excerpt',
						excerptLength: DEFAULT_EXCERPT_LENGTH,
```

We need to change the value of `excerptLength` to the _default_ value. It's a good idea to define anew constant for the default.